addons/website_event/controllers/main.py

```diff
@@ -42,10 +42,14 @@ def _get_events_search_options(self, **post):
 
     @http.route(['/event', '/event/page/<int:page>', '/events', '/events/page/<int:page>'], type='http', auth="public", website=True, sitemap=sitemap_event)
     def events(self, page=1, **searches):
-        if searches.get('tags') and request.httprequest.method == 'GET' and not searches.get('prevent_redirect'):
+        if len(searches.get('tags', '[]')) > 3 and request.httprequest.method == 'GET' and not searches.get('prevent_redirect'):
```

Comment:
Search for number of commas
```suggestion
        if searches.get('tags', '[]').count(',') > 0 and request.httprequest.method == 'GET' and not searches.get('prevent_redirect'):
```